src/axiosConfig.js

```diff
@@ -6,7 +6,14 @@ const axiosConfig = axios.create({
   baseURL: REACT_APP_BASE_URL,
   headers: {
     'content-type': 'application/json',
-    authorization: `Bearer ${localStorage.token}`,
   },
 });
 export default axiosConfig;
```

Comment:
Please export this at the bottom of the file @BrianSerem

Reply:
I have exported this at the bottom, any more changes you need to be made?

Reply:
No